docs/database.md

```
@@ -243,12 +243,47 @@ CREATE TABLE quiz_sessions (
   total_questions  INT NOT NULL DEFAULT 0,
   correct_count    INT NOT NULL DEFAULT 0,
   score_percentage NUMERIC(5,2) NULL,
+  time_limit_seconds INT NULL,          -- exam countdown duration (NULL for study mode)
+  passed           BOOLEAN NULL,        -- score >= pass_mark (NULL for study/incomplete)
   created_at       TIMESTAMPTZ NOT NULL DEFAULT now(),
   deleted_at       TIMESTAMPTZ NULL     -- added in migration 023 for discarded sessions
   -- No updated_at: only ended_at is set once, on completion
 );
 ```
 
+### exam_configs
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)